### ISSUE TITLE
Does not work after Bobcat Update

### ISSUE THREAD

**Issue by DaniDD** [aidanmelen/bobcat-miner-python#38]
I get an error:
AttributeError: 'NoneType' object has no attribute 'upper'

**Comment by aidanmelen** [comment]
Yeah I am not surprised it is broken now. Bobcat devs removed the status endpoint so there no longer an API to retrieve the miner's blockchain gap... it kinda sucks they did that.

**Comment by aidanmelen** [comment]
Looks like they also removed fast sync.
